makefiles/Makefile.third_party.win.mk

```diff
@@ -17,6 +17,9 @@ SOPLEX_TAG = 2.2.0
 # Version of Sulum
 SULUM_TAG = 43
 
+# Added in support of clean third party targets
+TSVNCACHE_EXE = TSVNCache.exe
```

Comment:
so I would check if it exists, and then fill in the cleaning macro accordingly.

Alternatively, We add the cleaning macro as a manual step, and you include that in your build process.